client/app/scripts/charts/nodes-chart.js

```diff
@@ -402,7 +394,6 @@ function mapStateToProps(state) {
   return {
     adjacentNodes: getAdjacentNodes(state),
     forceRelayout: state.get('forceRelayout'),
-    nodes: state.get('nodes').filter(node => !node.get('filtered')),
```

Comment:
Why not keep it there? The parent component does not need `nodes`. `layoutPrecision` can be calculated here too.